src/layers/core/point-density-hexagon-layer/point-density-hexagon-layer.js

```diff
@@ -48,13 +53,10 @@ function _needsReProjectPoints(oldProps, props) {
   return oldProps.radius !== props.radius;
 }
 
-function _getCountRange(hexagons) {
-  return [
-    Math.min.apply(null, hexagons.map(bin => bin.points.length)),
-    Math.max.apply(null, hexagons.map(bin => bin.points.length))
-  ];
+function _percentilChanged(oldProps, props) {
```

Comment:
My thinking is any method that doesn't require the `this` context can be a static method that's outside the layer